site/src/components/pages/dashboard/utils.ts

```diff
@@ -8,3 +18,129 @@ export const dashboardPages: { tabTitle: string; tabHref: string }[] = [
     tabHref: "/settings/api-keys",
   },
 ];
+
+export type DashboardSection = "create" | "manage" | "explore";
+
+export const getDashboardSectionCards = (props: {
+  openCreateSchemaModal: () => void;
+  profileLink: string;
+  userHasApiKey?: boolean;
+}): Record<DashboardSection, DashboardCardProps[]> => {
+  return {
+    create: [
+      {
+        title: "Build a block",
+        colorGradient:
+          "linear-gradient(291.34deg, #FFB172 -4.12%, #D082DE 53.49%, #9482FF 91.74%, #84E6FF 151.68%)",
+        colorGradientOnHover:
+          "linear-gradient(292.67deg, #FFB172 44.38%, #D082DE 83.44%, #9482FF 109.38%, #84E6FF 150.02%)",
+        description:
+          "Read the quickstart guide and learn how to create and use blocks",
+        link: {
+          title: "Read the guide",
+          href: "/docs/developing-blocks",
+        },
+      },
+      {
+        title: "Publish a block",
+        colorGradient:
+          "linear-gradient(310.17deg, #FFB172 -167.67%, #9482FF 13.54%, #84E6FF 126.83%)",
+        colorGradientOnHover:
+          "linear-gradient(304.41deg, #FFB172 -167.57%, #9482FF -22.66%, #84E6FF 53.07%)",
+        description: "Build a block you’re ready to release on the hub?",
+        link: {
+          title: "Publish a block",
+          /** @todo link to block publishing flow */
+          href: `${props.profileLink}/blocks`,
+        },
+      },
+      {
+        title: "Create a Type",
+        colorGradient:
+          "linear-gradient(91.21deg, #FFB172 -84.62%, #9482FF 62.56%, #84E6FF 154.58%)",
+        colorGradientOnHover:
+          "linear-gradient(91.21deg, #FFB172 -84.54%, #9482FF 27.8%, #84E6FF 98.03%)",
+        description:
+          "Types are a formal way to describe data, links, properties and entities",
+        link: {
+          title: "Create a type",
+          onClick: props.openCreateSchemaModal,
+        },
+      },
+    ],
+    manage: [
+      {
+        title: props.userHasApiKey
+          ? "Create and manage API keys"
+          : "Create your first API key",
+        description:
+          "Your API key will allow you to search for and blocks by name, author, or compatible data structure",
+        link: {
+          title: "Manage keys",
+          href: "/settings/api-keys",
+        },
+        icon: faKey,
+        variant: "secondary",
+      },
+      {
+        /** @todo show "View your public profile" if user is already completed profile */
+        title: "Complete your public profile",
```

Comment:
Since we don't have the code in for "Completing profile", let's use "View your public profile" and add a `@todo` to switch back when update profile functionality is in

Reply:
Updated at https://github.com/blockprotocol/blockprotocol/pull/511/commits/b0e108599e1954e95939a7b15acce487c24f45fd